docs/DeveloperGuide.adoc

```diff
@@ -317,10 +319,11 @@ The following activity diagram summarize what happens inside the `UndoRedoStack`
 image::UndoRedoActivityDiagram.png[width="650"]
 
 ==== Design Considerations
+This section highlights the implementation consideration of the program.
```

Comment:
`This section highlights the considerations behind our app's design choices` sounds better